docs/source/changelog.rst

```diff
@@ -26,17 +26,37 @@ We strongly recommend that you upgrade to version 9+ of pip before upgrading ``n
 5.2.0
 -----
 
-- Make all files in the dashboard editable by default and provide a whitelist of viewable file extensions (:ghpull:`2911`).
-- Ensure that the root directory is not hidden (:ghpull:`2907`).
-- Fix CodeMirror styling issue (:ghpull:`2880`).
-- Update error handling on ``APIHandlers`` (:ghpull:`2853`).
-- Upgrade xterm.js to 2.9.2 (:ghpull:`2849`).
-- Add Chinese translations (depends on upcoming UI implementation) (:ghpull:`2804`).
-- Allow for simpler customization of ``output_prompt`` in custom.js (:ghpull:`2774`).
-- Use RFC5987 encoding for filenames (:ghpull:`2767`).
-- Add path to the resources metadata (:ghpull:`2753`).
-- Make ``extraKeys`` consistent between notebook and editor views (:ghpull:`2745`).
-- Add RTL (right-to-left) support (:ghpull:`2357`).
+- Allow setting token via jupyter_token env (#2921). 
```

Comment:
Can you do a find & replace on these to use the `:ghpull:` rst role (like they were before this PR)? That way they automatically get links to the relevant PRs on Github.